esp-hal/src/spi/slave.rs

```diff
@@ -91,16 +93,17 @@ const MAX_DMA_SIZE: usize = 32768 - 32;
 /// SPI peripheral driver.
 ///
 /// See the [module-level documentation][self] for more details.
+#[instability::unstable]
```

Comment:
given the whole module is unstable this is just to make the docs look nicer, right? (just in case I am missing something important here)

Reply:
That, and we'll have to explicitly go through these once we commit to stabilizing SPI slave. FWIW rustdoc tends to ignore these, too, in places.